src/ui/server/test-mocks/mock-business-contact.ts

```diff
@@ -0,0 +1,17 @@
+import dotenv from 'dotenv';
+import FIELD_IDS from '../constants/field-ids/insurance/business';
+import ACCOUNT_FIELD_IDS from '../constants/field-ids/insurance/account';
+
+dotenv.config();
+
+const { POSITION } = FIELD_IDS.CONTACT;
+const { FIRST_NAME, LAST_NAME, EMAIL } = ACCOUNT_FIELD_IDS;
+
+const mockBusinessContact = {
+  [FIRST_NAME]: 'firstName',
+  [LAST_NAME]: 'lastName',
```

Comment:
Not a blocker but can we call these anything other than the actual IDs themselves 🙃